test_src/sljitTest.c

```diff
@@ -11571,6 +11577,18 @@ static void test92(void)
 	sljit_sw buf[45];
 	sljit_s32 i;
 
+/* Check whether the processor supports LoongArch V1.10's AMCAS instruction*/
```

Comment:
I am not happy about this direction. Maybe we should introduce a feature for atomics, and if it is not available, skip the test. Then the large ifdef could go away.

Reply:
Just like the SLJIT_HAS_SIMD I think? This maybe a good idea for both side.